src/Components/Components/src/PublicAPI.Shipped.txt

```diff
@@ -310,7 +310,7 @@ readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeDiff.ComponentId -
 readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeDiff.Edits -> Microsoft.AspNetCore.Components.RenderTree.ArrayBuilderSegment<Microsoft.AspNetCore.Components.RenderTree.RenderTreeEdit>
 readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeEdit.MoveToSiblingIndex -> int
 readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeEdit.ReferenceFrameIndex -> int
-readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeEdit.RemovedAttributeName -> string!
+readonly Microsoft.AspNetCore.Components.RenderTree.RenderTreeEdit.RemovedAttributeName -> string?
```

Comment:
Could you add this to https://github.com/aspnet/Announcements/issues/444?

Reply:
Done.